### ISSUE TITLE
about the loss function

### ISSUE THREAD

**Issue by LiMinghui19** [zzr-idam/4KDehazing#6]
Hi, thanks for your contribution.
when I read your paper, you use l2 loss as your loss function.
However, in your code, it seems like you used l1 loss, am I right?

Train.py：(line 25)
mse = nn.L1Loss().cuda()

so, the result in your paper is obtained by L1 or L2 loss?

**Comment by zzr-idam** [comment]
In the paper, we use the l2 loss function as targer, but  effect of the l1 is good.
